.gitignore

```diff
@@ -40,4 +40,7 @@ jspm_packages
 # Optional REPL history
 .node_repl_history
 
-.env*
\ No newline at end of file
+# VsCode Settings
+.vscode/
```

Comment:
philosophically, I think these kinds of "my environment" ignores should be in your own system dotfiles, not in the project's.  A la https://jaketrent.com/post/gitignore-system-files/